### DISCUSSION TITLE
Mender for Jetson Linux release: R35.2.1 JetPack release: 5.1.0

### DISCUSSION THREAD

**Discussion by marius-westgaard** [OE4T/meta-tegra#1218] [Q&A] [answered]
Hi,

According to the notes from the monthly meeting [02-09](https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2023-02-09) the meta-tegra team is looking into supporting Mender for the newest kirkstone release, with an expected update from Nvidia coming in March to facilitate A/B bootloader updates.

Are there any updated timelines on when support for the newest kirsktone release will be made available in meta-mender-community repository?

**Comment by dwalkes** [comment, marked as answer]
@marius-westgaard you can follow the discussion at https://matrix.to/#/#OE4T_jetpack-5-device-update:gitter.im to follow progress or, ideally, help implement if you need this feature.

**Reply by marius-westgaard** [reply]
Thanks for the reply! I'll follow the discussion and see if there is anything we can do.
